netctl/netctl.go

```diff
@@ -438,15 +535,16 @@ func createEndpointGroup(ctx *cli.Context) {
 	tenant := ctx.String("tenant")
 	network := ctx.Args()[0]
 	group := ctx.Args()[1]
+	netprofile := ctx.String("networkprofile")
```

Comment:
Just for my knowledge. Why do we need to specify netprofile while creating a network ? netprofile is already attached to the epg right ?